client/src/styles/legacy/_style.scss

```diff
@@ -1354,13 +1354,17 @@ form.member-profile-form {
 
 .cms-panel {
   .cms-panel-toggle {
+    display: flex;
+    justify-content: flex-end;
+    padding: 0;
 
     .toggle-collapse,
     .toggle-expand {
-      height: 54px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
```

Comment:
To vertically centre the `<span>&raquo;</span>`, without relying on `line-height` hacks